src/graph/Graph.js

```diff
@@ -558,9 +561,6 @@
         edgeElements.enter().append("g")
             .attr("class", "graphEdge")
             .style("opacity", 1e-6)
-            .on("click.selectionBag", function (d) {
-                context._selection.click(d, d3.event);
-            })
```

Comment:
idk i was getting something weird with that ... where it looked like i needed it to be removed .... but im going to put it back